src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -866,29 +874,136 @@ lemma thickening_subset_cthickening_of_le {δ₁ δ₂ : ℝ} (hle : δ₁ ≤ 
   thickening δ₁ E ⊆ cthickening δ₂ E :=
 (thickening_subset_cthickening δ₁ E).trans (cthickening_mono hle E)
 
-lemma cthickening_eq_Inter_cthickening {δ : ℝ} {E : set α} (δ_nn : 0 ≤ δ) :
-  cthickening δ E = ⋂ (ε : ℝ) (h : δ < ε), cthickening ε E :=
+lemma thickening_subset_interior_cthickening (δ : ℝ) (E : set α) :
+  thickening δ E ⊆ interior (cthickening δ E) :=
+(subset_interior_iff_open.mpr (is_open_thickening)).trans
+  (interior_mono (thickening_subset_cthickening δ E))
+
+lemma closure_thickening_subset_cthickening (δ : ℝ) (E : set α) :
+  closure (thickening δ E) ⊆ cthickening δ E :=
+(closure_mono (thickening_subset_cthickening δ E)).trans is_closed_cthickening.closure_subset
+
+/-- The closed thickening of a set contains the closure of the set. -/
+lemma closure_subset_cthickening {δ : ℝ} (δ_nn : 0 ≤ δ) (E : set α) :
```

Comment:
The assumption of nonnegativity is not needed here. I have the following in a branch
```lean
lemma cthickening_of_nonpos {δ : ℝ} (hδ : δ ≤ 0) (E : set α) :
  cthickening δ E = closure E :=
by { ext x, simp [mem_closure_iff_inf_edist_zero, cthickening, ennreal.of_real_eq_zero.2 hδ] }

@[simp] lemma cthickening_zero (E : set α) : cthickening 0 E = closure E :=
cthickening_of_nonpos le_rfl E

lemma closure_subset_cthickening (δ : ℝ) (E : set α) :
  closure E ⊆ cthickening δ E :=
by { rw ← cthickening_of_nonpos (min_le_right δ 0), exact cthickening_mono (min_le_left δ 0) E }
```
Would you merge this in your PR?

Reply:
Of course, thank you! I will do that later this week --- for now I change to "awaiting-author".